src/components/LoansByCategory/FeaturedHeroLoan.vue

```diff
@@ -0,0 +1,385 @@
+<template>
+	<div class="featured-section-wrapper">
+		<div class="row">
+			<div class="column small-12">
+				<h2 class="section-name">Featured: Research-backed impact</h2>
+			</div>
+		</div>
+		<div class="featured-row-wrapper">
+			<div class="featured-cards-display-window">
+				<div class="grid-loan-card">
+					<div class="row featured-hero-loan">
+						<div class="column small-12 medium-5">
+							<loan-card-image
+								:loan-id="loan.id"
+								:name="loan.name"
+								:retina-image-url="loan.image.retina"
+								:standard-image-url="loan.image.default"
+								:is-visitor="!isLoggedIn"
+								:is-favorite="loan.userProperties.favorited"
+								:image-enhancement-experiment-version="imageEnhancementExperimentVersion"
+								:loan-image-hash="loan.image.hash"
+
+								@track-loan-card-interaction="trackInteraction"
+								@favorite-toggled="toggleFavorite"
+							/>
+						</div>
+						<div class="column small-12 medium-7">
+							<div class="borrower-info-wrapper">
+								<div class="name">
+									<router-link
+										:to="`/lend/${loan.id}`"
+										v-kv-track-event="['Lending', 'click-Read more', 'Name', loan.id, 'true']">
+										<span
+											@click="$emit('track-loan-card-interaction', {
+												interactionType: 'viewBorrowerPage',
+												interactionElement: 'borrowerName'
+											})"
+										>{{ loan.name }}</span>
+									</router-link>
+								</div>
+
+								<div class="country">
+									<!-- eslint-disable-next-line max-len -->
+									{{ loan.geocode.country.name }} <span v-if="loan.activity.name">/ {{ loan.activity.name }}</span>
+								</div>
+
+								<div class="use">
+									<div class="show-for-large">
```

Comment:
I think you can combine the 2 divs that show "loanUse" and work out a combination of classes to apply to get that working.